lib/virtual-fs/src/webc_volume_fs.rs

```diff
@@ -295,21 +293,37 @@ impl AsyncWrite for File {
     }
 }
 
+// HACK: timestamps are not present in webc v2, so we have to return
+// a stub modified time. previously we used to just return 0, but that
+// proved to cause problems with programs that interpret the value 0.
+// to circumvent this problem, we decided to return a non-zero value.
+fn get_modified(timestamps: Option<webc::Timestamps>) -> u64 {
+    static DEFAULT: OnceLock<u64> = OnceLock::new();
+
+    timestamps
+        .map(|t| t.modified())
```

Comment:
Doesn't this introduce somewhat weird random behaviour?

Why not just use `1` instead?

Reply:
I'm also fine with returning `1`.